docs/user-documentation/CLAWfor1x.md

```diff
@@ -32,10 +32,10 @@ In Fedora 4 , what we would have called `objects` are now referred to as `resour
 * `Resource`: Roughly equivalent to a Fedora 3 object - a conceptual representation of a thing that can contain files or other containers.
 * `Non-RDF Source`: Roughly equivalent to a datastream. A Non-RDF Source (or binary) is simply a bitstream (e.g. JPG, PDF, XML, MP3, etc.).
 
-###Datastreams
-In Islandora CLAW, RDF datastreams (RELS-EXT and RELS-INT) are stored as RDF in Fedora. Binary datastreams (files, images) are files or `nonRdfResources` (see [PCDM](https://github.com/duraspace/pcdm/wiki)). Descriptive metadata datastreams (MODS, DC, DwC, PBCore, etc) are will be stored as RDF.
+### Datastreams
```

Comment:
@ruebot Ah, I misunderstood what you meant. I thought you meant it like "are stored as [RDF|`RDFSource`]" so I picked one. Yet another incoming commit.